lib/jekyll-seo-tag/json_ld_drop.rb

```diff
@@ -79,8 +79,22 @@ def main_entity
       alias_method :mainEntityOfPage, :main_entity
       private :main_entity
 
+      private def deep_merge(this_hash, other_hash)
```

Comment:
Two issues here:
- Jekyll Core already has a method for similar task: `Jekyll::Utils.deep_merge_hashes`. Why re-invent the wheel?
- This code base doesn't use inline private calls for explicit method definitions. Private methods are defined at the bottom of a class under the `private` keyword.

Reply:
Ah, I see. I wasn't aware of that builtin function, so I've now removed the custom one and changed it to use Jekylls version instead.